### PR TITLE
Changelogs for Mender 3.5.1.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

File: 10.Mender-server/docs.md [hunk 1]
```diff
@@ -8,6 +8,98 @@ github: false
...
+### Changelogs
+
+#### deployments (4.4.1)
+
+New changes in deployments since 4.4.0:
+
+##### Bug Fixes
+
+* Use separate object storage path for raw file and artifact
+  ([MEN-6091](https://tracker.mender.io/browse/MEN-6091))
```
Comment (0lmi) [inline 10.Mender-server/docs.md:56]: Do we want to keep links to the old Jira instance here?

Comment (kacf) [reply 10.Mender-server/docs.md:56]: Good catch! No, we don't. I'll fix that, and the tool that generated it as well.

**Comment by kacf** [comment]
@0lmi: Now it should be good!

**Review by 0lmi — APPROVED** [review]
👍

**Pull request merged**